src/engine/tools/vagrant/Vagrantfile

```diff
@@ -171,6 +182,7 @@ Vagrant.configure("2") do |config|
         echo "{}" > /var/ossec/etc/engine-test.conf
         engine-test add -n windows -f eventchannel
         engine-test add -n syslog -f syslog -o /tmp/syslog.log
+        engine-test add -n remote-syslog -f remote-syslog -o 127.0.0.1
```

Comment:
improve-desing: add command dont have -n parameter. it's change to -i

Reply:
done! nice catch!